ush/set_FV3nml_ens_stoch_seeds.py

```diff
@@ -58,7 +58,7 @@ def set_FV3nml_ens_stoch_seeds(cdate):
     #
     # -----------------------------------------------------------------------
     #
-    fv3_nml_ensmem_fp = f"{os.getcwd()}{os.sep}{FV3_NML_FN}"
+    fv3_nml_ensmem_fp = f"{os.getcwd()}{os.sep}{FV3_NML_FN}_base"
```

Comment:
The base namelist file has a `_base` extension and all others are derived from it, for example, appending stochastic physics parameters etc.